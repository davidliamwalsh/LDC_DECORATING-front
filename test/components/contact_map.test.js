import ContactMap from '../../components/ContactMap'

describe('<ContactMap />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <ContactMap />
    )

    expect(component).toMatchSnapshot()
  })
})