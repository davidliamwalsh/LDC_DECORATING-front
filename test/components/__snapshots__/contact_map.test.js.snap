// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContactMap /> it renders correctly 1`] = `ShallowWrapper {}`;
